examples/quick-start-workspace/flake.nix

```
@@ -120,6 +120,11 @@
             inherit src;
           };
 
+          my-workspace-toml-fmt = craneLib.taploFmt {
+            inherit src;
```

Might be good to highlight here that we can use a source filter to cut down the number of times the derivation needs to be rebuilt when _other_ source files change.

For example:

```suggestion
            src = pkgs.lib.sources.sourceFilesBySuffices src [ ".toml" ];
```